### ISSUE TITLE
<amp-date-picker> support for custom localization/translation

### ISSUE THREAD

**Issue by SimonHogstromRonbol** [ampproject/amphtml#26654]
## Describe the new feature or change to an existing feature you'd like to see

The amp-date-picker is amazing and does everything we need, the problem is however that i'm building a site in danish, and the weekdays and and months, along with the info at the little questionmark is all in english, and i have been unable to find a way to translate it.

i know that danish isn't a supported language for amp so i would like to be able to do custom translations, maybe this could be part of the template system that the datepicker is already using.

## Describe alternatives you've considered

I've considered just using the english version on my danish site, most danes have an alright grasp of the english language and is used to having english phrases as part of the ui when browsing the web, my client in this particular case is an ensurance company, and the form im building is for filing reports, a lot of trust goes into filing this kind of infomation, so i'm afraid that having the ui in different languages might reduce the level of trust that users put into the site.

another alternative is building my own date picker, this would be a huge waste of time, both for me building it, and for the many developers who've spent time building <amp-date-picker> to begin with.

## Additional context
here is a link directly to the amp-date-picker playground:
https://playground.amp.dev/?url=https%3A%2F%2Fpreview.amp.dev%2Fdocumentation%2Fexamples%2Fcomponents%2Famp-date-picker&format=websites

@morsssss

**Comment by caroqliu** [comment]
`amp-date-picker` uses `react-dates` which [supports localization](https://github.com/airbnb/react-dates#localization). You can add `locale="da"` on your `<amp-date-picker>` element to solve the date localization piece of this issue.

In terms of the additional information in the question mark -- it looks like react-dates also supports this through passing in a `phrases` prop. In order to support this, the `amp-date-picker` element will need to look for that prop in order to pass it through. Since it is more complicated than a string, we will probably want to pass it in through some type of configuration. Another option is to hide the panel completely with the `hide-keyboard-shortcuts-panel` attribute.

**Comment by SimonHogstromRonbol** [comment]
@caroqliu this is fantastic news!

I gave it a try and it works like a charm, thank you.
https://codepen.io/sunturion/pen/bGddYrE

Maybe the locale attribute should be part of the [amp.dev](https://amp.dev/documentation/components/amp-date-picker/) documentation.

**Comment by caroqliu** [comment]
Glad to hear it worked out!! 😄Closing the issue now.

BTW, it looks like the [`locale` attribute](https://amp.dev/documentation/components/amp-date-picker/#locale) is in the documentation, though I agree the language could be updated to make it easier to find and use.

**Comment by SimonHogstromRonbol** [comment]
@caroqliu you're right, totally missed that, i searched the doc for language, localization, translation and even i18n :)
